From ff2b8e6e029d4fa1079189aa74a0e2a03acadd38 Mon Sep 17 00:00:00 2001
From: Hugues Fruchet <hugues.fruchet@st.com>
Date: Fri, 29 Jul 2016 17:03:52 +0200
Subject: [PATCH 06/23] v4l2videodec: increase ranking

Allows to select this element for video hardware acceleration
regarding any other proprietary plugins that could have
PRIMARY + 1 ranking.

Change-Id: I9f56b747497c2724253a1f475f2d73b1be2f7b47
Reviewed-on: https://gerrit.st.com/56796
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
Tested-by: Hugues FRUCHET <hugues.fruchet@st.com>
---
 sys/v4l2/gstv4l2videodec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/v4l2/gstv4l2videodec.c b/sys/v4l2/gstv4l2videodec.c
index 8ca5e38..59a43cd 100644
--- a/sys/v4l2/gstv4l2videodec.c
+++ b/sys/v4l2/gstv4l2videodec.c
@@ -1027,7 +1027,7 @@ gst_v4l2_video_dec_register (GstPlugin * plugin, const gchar * basename,
   type_name = g_strdup_printf ("v4l2%sdec", basename);
   subtype = g_type_register_static (type, type_name, &type_info, 0);
 
-  gst_element_register (plugin, type_name, GST_RANK_PRIMARY + 1, subtype);
+  gst_element_register (plugin, type_name, GST_RANK_PRIMARY + 2, subtype);
 
   g_free (type_name);
 
-- 
2.7.4

